packages/medusa/src/api/store/shipping-options/[id]/calculate/route.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpTypes } from "@medusajs/framework/types"
+import { MedusaRequest, MedusaResponse } from "@medusajs/framework/http"
+import { calculateShippingOptionsPricesWorkflow } from "@medusajs/core-flows"
+
+import { StoreCalculateShippingOptionPriceType } from "../../validators"
+
+export const POST = async (
+  req: MedusaRequest<StoreCalculateShippingOptionPriceType>,
```

Comment:
Can we use an HTTP type here instead? It'll be used by the JS SDK at a later point too.